### PR TITLE
Video-41-Delete-Product

### PR DESCRIPTION
**Pull request by basir**

### CODE REVIEW

File: frontend/src/actions/productActions.js [hunk 1]
```diff
@@ -84,3 +87,21 @@ export const updateProduct = (product) => async (dispatch, getState) => {
...
+export const deleteProduct = (productId) => async (dispatch, getState) => {
+  dispatch({ type: PRODUCT_DELETE_REQUEST, payload: productId });
+  const {
+    userSignin: { userInfo },
+  } = getState();
+  try {
+    const { data } = Axios.delete(`/api/products/${productId}`, {
+      headers: { Authorization: `Bearer ${userInfo.token}` },
+    });
+    dispatch({ type: PRODUCT_DELETE_SUCCESS });
```
Comment (Nolegal) [inline frontend/src/actions/productActions.js:99]: 99str, payload: data

Comment (payanazo) [reply frontend/src/actions/productActions.js:99]: was looking for this

File: frontend/src/screens/ProductListScreen.js [hunk 2]
```diff
@@ -16,16 +23,30 @@ export default function ProductListScreen(props) {
     success: successCreate,
     product: createdProduct,
   } = productCreate;
+
+  const productDelete = useSelector((state) => state.productDelete);
+  const {
+    loading: loadingDelete,
```
Comment (sanketwagh511) [inline frontend/src/screens/ProductListScreen.js:29]: TypeError: Cannot destructure property 'loading' of 'menuDelete' as it is undefined.

Comment (marothi-codes) [reply frontend/src/screens/ProductListScreen.js:29]: check your productDeleteReducer in productReducers.js and ensure that there's no typo on `return { loading: true/false }`...

File: frontend/src/actions/productActions.js (hunk 1 above)
Comment (Aron-Bijl) [reply frontend/src/actions/productActions.js:99]: It should be:     try {
      await axios.delete(`/api/products/${productId}`, {
        headers: { Authorization: `Bearer ${userInfo.token}` },
      });

Comment (fahmiyuseri) [reply frontend/src/actions/productActions.js:99]: a

**Pull request merged**